tests/rptest/tests/restart_services_test.py

```diff
@@ -43,31 +37,24 @@ def __init__(self, context, **kwargs):
             **kwargs)
 
     @cluster(num_nodes=3)
-    def test_restart_services(self):
+    def test_restart_services_failures(self):
         admin = Admin(self.redpanda)
 
         # Failure checks
         self.logger.debug("Check restart with no service name")
         try:
-            admin.redpanda_services_restart()
+            admin.restart_service()
         except requests.exceptions.HTTPError as ex:
             self.logger.debug(ex)
-            assert ex.response.status_code == requests.codes.bad_request
+            assert ex.response.status_code == requests.codes.not_found
```

Comment:
I'm wondering if this will go back to `bad_request` when you add the parameter back into the json file